Trouble running tests locally Ubuntu 14.04

Seem to be having docker issues.

test-integration.sh output with latest Ubuntu release of docker
(https://docs.docker.com/installation/ubuntulinux/#ubuntu-trusty-1404-lts-64-bit)

```
--- FAIL: TestBuilder (5.00 seconds)
    dockercli.go:266: Error: client and server don't have same version (client : 1.16, server: 1.12)
```

test-integration.sh output with very latest docker release ( lxc-docker 1.3.2)
(https://docs.docker.com/installation/ubuntulinux/#ubuntu-trusty-1404-lts-64-bit)

```
make[1]: Entering directory `/home/jparker/workspaces/deis/router'
docker build -t router-build parent
Sending build context to Docker daemon 357.4 kB
Sending build context to Docker daemon 
Step 0 : FROM golang:1.3
 ---> f915f8f4eea6
Step 1 : ADD build.sh /go/src/github.com/deis/deis/router/build.sh
 ---> Using cache
 ---> 956e63f1f3f4
Step 2 : WORKDIR /go/src/github.com/deis/deis/router
 ---> Using cache
 ---> 78ec4e56b5a0
Step 3 : RUN DOCKER_BUILD=true /go/src/github.com/deis/deis/router/build.sh
 ---> Using cache
 ---> cdd4f0897cd0
Step 4 : RUN tar -C /opt/nginx -czf /nginx.tgz .
 ---> Using cache
 ---> d8bef1b5e029
Step 5 : RUN go get github.com/tools/godep
 ---> Using cache
 ---> 74033148d8df
Step 6 : ADD . /go/src/github.com/deis/deis/router/
 ---> Using cache
 ---> 0bb4f60371f4
Step 7 : RUN CGO_ENABLED=0 godep go build -a -ldflags '-s' boot.go
 ---> Using cache
 ---> 286f62c5ddfc
Step 8 : RUN cp /go/src/github.com/deis/deis/router/boot /go/bin/boot
 ---> Using cache
 ---> 8eb387f1b4f3
Successfully built 8eb387f1b4f3
docker cp `docker run -d router-build`:/nginx.tgz .
2014/11/25 15:22:35 "nginx.tgz" is outside of "."
```

Any ideas?

--- a/router/Makefile
+++ b/router/Makefile
@@ -7,7 +7,7 @@ BUILD_IMAGE = $(COMPONENT)-build
 
 build: check-docker
 	docker build -t $(BUILD_IMAGE) parent
-	docker cp `docker run -d $(BUILD_IMAGE)`:/nginx.tgz .
+	docker cp `docker run -d $(BUILD_IMAGE)`:/nginx.tgz `pwd`
 	docker cp `docker run -d $(BUILD_IMAGE)`:/go/bin/boot bin/
 	docker build -t $(IMAGE) .
 	rm nginx.tgz

